### DISCUSSION TITLE
自由地在 Show And Tell 中，分享你生成的春联～

### DISCUSSION THREAD

**Discussion by YunYouJun** [YunYouJun/ai-sfc#3] [Announcements]
自由地在 Show And Tell 中，分享你生成的春联～

**Comment by Big-Cake-jpg** [comment]
![91e70cd62a848aa4b767577a95462eed](https://github.com/YunYouJun/ai-sfc/assets/76810494/4d8eebd8-6265-4fe9-96dd-707ef5c978f8)
